web_development_101/javascript_basics/project_etch_a_sketch.md

```diff
@@ -43,8 +43,6 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
   <summary> Show Student Solutions </summary>
 
 - Add your solution below this line!
-* [Nick DiPreta's solution](https://github.com/NickDiPreta/etch-a-sketch) - [View in Browser](https://nickdipreta.github.io/etch-a-sketch/)
-* [justasdap Solution](https://github.com/JustasDap/etchasketch) - [View in Browser](https://htmlpreview.github.io/?https://github.com/JustasDap/etchasketch/blob/master/index.html)
```

Comment:
Why did you remove these?

Reply:
Oops I didn't mean to do that... First time I'm contributing to an open source, it is a mistake... I just followed the video. 
Should I redo the process ? 
So sorry about that, that was not intended at all. I will pay more attention next time.

Reply:
> Oops I didn't mean to do that... First time I'm contributing to an open source, it is a mistake... I just followed the video.
> Should I redo the process ?
> So sorry about that, that was not intended at all. I will pay more attention next time.

Yes, I will close this, you can just create a new one.